karaf/init/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -44,6 +44,7 @@ limitations under the License.
             <cm:property name="persistPeriod" value="1s" />
             <cm:property name="globalBrooklynPropertiesFile" value="~/.brooklyn/brooklyn.properties" />
             <cm:property name="localBrooklynPropertiesFile" value="" /> <!-- used to be settable through cli params -->
+            <cm:property name="default.catalog.location" value="etc/default.catalog.bom" />
```

Comment:
But it still relies on relative path? Would it makes sense to prefix with `${karaf.etc}`? Could see the etc folder relocated for rpm/deb?